test/integration_test_helper.rb

```diff
@@ -0,0 +1,123 @@
+ENV["RAILS_ENV"] = "test"
```

Comment:
Is minitest/autorun needed here?  I'm unclear about what it's doing in either case to be honest.